src/ontology/cl-edit.owl

```diff
@@ -31078,6 +31083,57 @@ SubClassOf(obo:CL_4030050 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0005382
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.cub.2021.10.015") obo:IAO_0000115 obo:CL_4030052 "A DRD2-expressing medium spiny neuron that is part of a nucleus accumbens shell or olfactory tubercle.")
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030052 "2023-06-14T13:27:51Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.cub.2021.10.015") oboInOwl:hasExactSynonym obo:CL_4030052 "NAc shell and OT D2 medium spiny neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.cub.2021.10.015") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4030052 "D2Sh")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.cub.2021.10.015") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4030052 "NAc shell and OT D2-MSN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.cub.2021.10.015") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4030052 "nucleus accumbens shell and olfactory tubercle D2-MSN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.cub.2021.10.015") rdfs:comment obo:CL_4030052 "In Rhesus macaques, both nucleus accumbens shell and olfactory tubercle D1 and D2 medium spiny neuron cell types have been noted to have enriched gene expression of the following: GREB1L, ARHGAP6, and GRIA4.")
+AnnotationAssertion(rdfs:label obo:CL_4030052 "nucleus accumbens shell and olfactory tubercle D2 medium spiny neuron")
+SubClassOf(obo:CL_4030052 obo:CL_1001474)
+SubClassOf(obo:CL_4030052 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0005403))
+SubClassOf(obo:CL_4030052 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001177))
+
+# Class: obo:CL_4030053 (Island of Calleja granule cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.cub.2021.10.015") obo:IAO_0000115 obo:CL_4030053 "A DRD1-expressing, medium spiny neuron-like granule cell that is part of an Island of Calleja.")
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030053 "2023-06-14T13:37:45Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.cub.2021.10.015") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4030053 "D1-ICj")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/j.cub.2021.10.015") rdfs:comment obo:CL_4030053 "In Rhesus macaques, the Island of Calleja granule cell type has been noted to have enriched gene expression of CPNE4.")
+AnnotationAssertion(rdfs:label obo:CL_4030053 "Island of Calleja granule cell")
+SubClassOf(obo:CL_4030053 obo:CL_0000120)
+SubClassOf(obo:CL_4030053 ObjectSomeValuesFrom(obo:RO_0002100 ObjectIntersectionOf(obo:UBERON_0001881 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005403))))
```

Comment:
This article indicates that [granule cells of ICj project to allocortical structures such as the septum, piriform cortex and amygdala](https://onlinelibrary.wiley.com/doi/epdf/10.1002/cne.901810209). Therefore, I would use `has soma location`, instead of `part of`.

Reply:
<img width="293" alt="Screenshot 2023-06-16 at 10 30 40" src="https://github.com/obophenotype/cell-ontology/assets/95643797/a1b59de2-6f26-487c-9dc2-93ae50c178d9">

The soma location for this cell type is in some island of Calleja that is **part of** some ventral striatum.
This modelling is based on [feedback here](https://github.com/obophenotype/cell-ontology/pull/2005#issuecomment-1580253089).

Reply:
Ok, I see. Sorry, I haven't come across many anonymous classes yet, so it is good to see an example.